sdk/storage/Azure.Storage.Files.Shares/src/Models/ShareCreateOptions.cs

```diff
@@ -64,5 +64,18 @@ public class ShareCreateOptions
         ///  Default if not specified is the maximum throughput the file share can support. Current maximum for a file share is 10,340 MiB/sec.
         /// </summary>
         public long? PaidBurstingMaxBandwidthMibps { get; set; }
+
+        /// <summary>
+        /// Optional.  Only applicable to provisioned v2 storage accounts.
+        /// The provisioned IOPS of the share.  For SSD, min IOPS is 3,000 and max is 100,000.  For HDD, min IOPS is 500 and max is 50,000.
```

Comment:
NIT - spell out min to minimum and max to maximum.

But also maybe in the future we can reference the docs instead of putting the limits here, in case the limits change one day, and we don't know about it and our docs potentially become incorrect in the future.

same for the XML summary below

Reply:
Fixed.